src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -515,9 +515,9 @@ object SonarLintIntelliJClient : SonarLintRpcClientDelegate {
         }
         val connection = connectionOpt.get()
         return if (connection.token != null) {
-            Either.forLeft(TokenDto(connection.token))
+            Either.forLeft(TokenDto(connection.token!!))
```

Comment:
Can they be null or are we handling the exception if they are?

Reply:
Normally it should not happen in our Wizard, it should be either token or password/login

Reply:
I reverted login/password `!!`, for token it's fine as we still check before